ddtrace/contrib/mysqldb/patch.py

```diff
@@ -58,6 +58,6 @@ def patch_conn(conn, *args, **kwargs):
     pin = Pin(service="mysql", app="mysql", app_type="db", tags=tags)
 
     # grab the metadata from the conn
-    wrapped = TracedConnection(conn)
+    wrapped = TracedConnection(conn, pin)
```

Comment:
Let's use the kwarg:
```python
wrapped = TracedConnection(conn, pin=pin)
```

Also can you check other mysql integrations? do they have the same issue?
If yes, let's do the same thing you did here in this PR, so that we fix all of them at once.

Reply:
I tested the other 2 mysql integrations.
`pymysql` was working fine.
`mysql-connector` had the same issue.
I patched both to be safe, tell me if it's not relevant.